app/src/main/java/com/quran/labs/androidquran/ui/fragment/TabletFragment.java

```diff
@@ -319,16 +320,18 @@ protected void onPostExecute(RectF[] rect) {
           if (mRightImageView != null && mLeftImageView != null) {
             mRightImageView.setPageBounds(rect[0]);
             mLeftImageView.setPageBounds(rect[1]);
-            Context context = getContext();
-            if (mOverlayText && context != null) {
```

Comment:
+1! 
There is no point in checking if context is null. If it is null, then something is seriously broken beyond repair.